src/containers/treePage/index.tsx

```diff
@@ -0,0 +1,248 @@
+  font-weight: bold;
+  color: ${ DARK_GREEN }
+`;
+
+const EntryMessage = styled(Paragraph)`
+  display: inline;
+  text-align: center;
+  line-height: 0px;
+  color: ${TEXT_GREY};
+`;
+
+const dummyCare: TreeCare[] = [
+  {
+    date: "March 28th",
+    message: "Was cleared of waste and weeded"
+  },
+  {
+    date: "March 27th",
+    message: "Was cleared of waste and weeded"
+  },
+  {
+    date: "March 26th",
+    message: "Was cleared of waste and weeded"
+  },
+  {
+    date: "March 25th",
+    message: "Was cleared of waste and weeded"
+  },
+  {
+    date: "March 24th",
+    message: "Was cleared of waste and weeded"
+  },
+  {
+    date: "March 23rd",
+    message: "Was cleared of waste and weeded"
+  },
+]
+
+const dummyTree: SiteProps = {
+  siteId: 1,
+  blockId: 1,
+  lat: 2,
+  lng: 2,
+  city: 'Boston',
+  zip: '02115',
+  address: '100 Super Tree Way',
+  entries: [{
+    id: 1,
+    username: 'Speak for the trees user',
+    updatedAt: 100000000,
+    treePresent: true,
+    status: 'good',
+    genus: 'biggus treebus',
+    species: 'larggeus',
+    commonName: 'large tree', 
+    confidence: 'good',
+    diameter: 1000000,
+    circumference: 3140000,
+    coverage: 'leaves',
+    discoloring: true,
+    pooling: false,
+    light: true,
+    bicycle: true,
+    fence: 'yes',
+    treeNotes: 'I think it looks pretty good',
+    siteNotes: 'dirty',
+  }]
+};
+
+const TreePage: React.FC<SiteProps> = ({
+  address,
+  entries
+}) => {
+
+  const latestEntry: SiteEntry = entries[entries.length - 1];
+
+  const description = "Dreamcatcher kogi taiyaki keytar. Swag typewriter craft beer cronut pok pok gentrify flannel salvia deep v pork belly"
+
+  return(
+    <>
+      <Helmet>
+        <title>Tree</title>
+        <meta
+          name="Individual Tree Page"
+          content="Description of a particular tree site, allows users to adopt a tree and monitor their stewardship activities."
+        />
+      </Helmet>
+      <PageLayout>
+        <TreePageContainer> 
+          {/*Change to tree map route once that page is finished*/}
+          <ReturnButton to={Routes.HOME}> 
+              {`<`} Return to Tree Map
+          </ReturnButton> 
+          <TreeMainContainer>
+          <Row>
+            <Col span={7}>
+              <TreeInfoContainer>
+                <TreeImage
+                  src={placeholder}
+                />
+              </TreeInfoContainer>
+            </Col>
+            <Col span={10}>
+              <TreeInfoContainer>
+                { latestEntry.commonName && 
+                  <PageHeader 
+                    pageTitle={ latestEntry.commonName }
+                  />
+                }
+                <Title level={2}>
+                  { address }
+                </Title>  
+                <Paragraph>
+                  { description }
+                </Paragraph>
+                <Button type="primary" size="large">
+                  Adopt
+                </Button>  
```

Comment:
Should be disabled if the user has already adopted this tree, maybe a redirect to login message if they're not logged in

Reply:
will set this up when connected to redux